### DISCUSSION TITLE
Ability to sort Album assets in IOS App

### DISCUSSION THREAD

**Discussion by findmarcusr** [immich-app/immich#10236] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The Albums seems to have a strange way of sorting. On the main photo page its great to see the newest first. However I find it odd that albums are sorted the same way. When I share albums with people and look at albums myself. Its seems better to view them in order they were taken. Using the web interface I am able to sort like this. But the iOS app does not seem to allow me to do this. Also when I share an album. The sorting should be shared as the sender intended, not always as newest to oldest. 

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#8734
